src/main.js

```diff
@@ -7,6 +7,9 @@ import { generateFilePath } from '@nextcloud/router'
 
 import Vue from 'vue'
 import App from './App.vue'
+import 'swagger-ui/dist/swagger-ui.css'
+import '../css/swagger-dark.css'
+import '../css/swagger-system.css'
```

Comment:
```suggestion
import '../css/swagger-theme-dark.css'
```
Import needs to be fixed

Reply:
Yes. I forgot the main.js